index.html

```diff
@@ -209,9 +210,87 @@ <h2><span id="circle"></span> Skills</h2>
         <li>Gitlab</li>
         <li>Terminal</li>
       </ul>
-    
+
     </div>
   </section>
+  <section class="contact-me">
+    <p>
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </p>
+    <form method="POST">
+      <ul class="form">
+        <li>
+          <input type="text" name="name" placeholder="Full name" maxlength="30" required />
+        </li>
+        <li>
+          <input type="email" name="email" placeholder="Email adress" required />
+        </li>
+        <li>
+          <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+        </li>
+        <li>
+          <button class="submit-btn" type="submit">Get in touch</button>

```

Comment:
- [ ] Please ensure you include a design for the button interaction states which are `hover`, `pressed`, `enabled` and `disabled`. Kindly checkout the design image below for directions

![image](https://user-images.githubusercontent.com/43995830/159655338-92633003-dd49-44f8-9b42-32d650cdecb9.png)